tests/log_parser/test_store_failure_lines.py

```diff
@@ -193,6 +194,26 @@ def test_store_error_summary_group_status(activate_responses, test_repository, t
     assert error_groups.first().name == "toolkit/components/pictureinpicture/tests/browser.ini"
 
 
+def test_group_status_duration(activate_responses, test_repository, test_job):
+    log_path = SampleData().get_log_path("mochitest-browser-chrome_errorsummary.log")
+    log_url = 'http://my-log.mozilla.org'
+
+    with open(log_path) as log_handler:
+        responses.add(responses.GET, log_url, body=log_handler.read(), status=200)
+
+    log_obj = JobLog.objects.create(job=test_job, name="errorsummary_json", url=log_url)
+    store_failure_lines(log_obj)
+
+    assert FailureLine.objects.count() == 5
+
+    ok_groups = Group.objects.filter(~Q(group_result__duration=0))
```

Comment:
The implementation logic guarantees the duration to be >= 0. `__gt` ([docs](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#gt)) should be used instead of `~Q`

Reply:
thanks!  this works